### PR TITLE
Tanasee142

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
This is my first attempt at one of the Tanasee mss., so if there are some peculiarities to this particular repository let me know. Thank you!

### CODE REVIEW

File: Tanasee/Tanasee142.xml [hunk 1]
```diff
@@ -26,16 +26,273 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+<title type="complete" ref="LIT2113OntheB"/>
+<note>Indication for reading: <q xml:lang="gez">ዘልደት፡ ምንባብ።</q></note>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3">
+<locus from="20v" to="33v"/>
+<title type="complete" ref="LIT2095OnHowO"/>
+<note><locus target="#21">F. 21</locus> is misbound. The text corresponds to what in some manuscripts is the final part of
+<ref type="work" corresp="LIT2118OntheD"/>, however here in <ref type="item" corresp="ms_i1.7"/> there is no loss of text.</note>
```
Comment (thea-m) [inline Tanasee/Tanasee142.xml:54]: For the internal ID, I would simply use `<ref target="#ms_i1.7"/>`

File: Tanasee/Tanasee142.xml [hunk 2]
```diff
@@ -26,16 +26,273 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+<item xml:id="a2">
+<locus from="2r" to="2v"/>
+<locus from="158r" to="158v"/>
+<desc type="GuestText"><ref type="work" corresp="LIT1960Mashaf"></ref></desc>
+<note>A wirting exercise. The text is incomplete.</note>
+</item>
+
+<item xml:id="a3">
+<locus target="#158r"/>
+<desc type="OwnershipNote">The note states that the manuscript belonged to the monastery of <placeName ref="LOC2422DagaEs">Dāgā ʾƎsṭifānos</placeName></desc>.
```
Comment (thea-m) [inline Tanasee/Tanasee142.xml:273]: we can use the institution ID INS0329DE since it definitely refers to this place as a repository of manuscripts

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! I don't think there is anything special to consider regarding the repository.

File: Tanasee/Tanasee142.xml (hunk 2 above)
Comment (MarcinKrawczuk) [reply Tanasee/Tanasee142.xml:273]: ok, will remember to use INS rather then geographical location

**Review by thea-m — APPROVED** [review]

**Pull request merged**